server/internal/usecase/interfaces/model.go

```diff
@@ -20,6 +20,11 @@ type CreateModelParam struct {
 	Key         *string
 	Public      *bool
 }
+type FindOrCreateSchemaParam struct {
```

Comment:
```suggestion

type FindOrCreateSchemaParam struct {
```

insert line break between struct